lib/config/app-strings.ts

```diff
@@ -5,6 +5,8 @@ export const configFileNames = [
   '.github/renovate.json5',
   '.gitlab/renovate.json',
   '.gitlab/renovate.json5',
+  '.gitea/renovate.json',
+  '.gitea/renovate.json5',
```

Comment:
@rarkins I'd like to change the platform specific files to only check for `.gitea/` on gitea and not on GitHub or gitlab, like renovate currently does.

so instead of adding all file paths here, we should process the platform specific files dynamically. like we already do for the organization onboarding preset.

Reply:
I agree